facterdb.gemspec

```
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.description = 'Contains facts from many Facter version on many Operating Systems'
   s.licenses    = 'Apache-2.0'
 
-  s.files       = `git ls-files`.split("\n")
+  s.files       = `git ls-files`.split("\n") + ['database.md']
```

I guess we do not need to bundle this file into the gem:

```suggestion
  s.files       = `git ls-files`.split("\n")
```

If I am wrong and we want to do so, it may make more sense to `require 'bundler/gem_tasks'` and change the Rake task to something like:

```
file Dir['facts/**/*.facts'] => 'database.md' do
```

that way, when we build with `rake build` the md file is updated if necessary before packaging the gem.

---

> I guess we do not need to bundle this file into the gem:

I wasn't sure: I think https://rubydoc.info/ generates the docs based on the gem, but that's something we should play with. Could even be a follow up step.

> If I am wrong and we want to do so, it may make more sense to `require 'bundler/gem_tasks'` and change the Rake task to something like:

Interesting idea. I'll think about that.